contracts/LSP8DigitalAsset.json

```diff
@@ -0,0 +1,593 @@
+{
+  "_format": "hh-sol-artifact-1",
+  "contractName": "LSP8IdentifiableDigitalAsset",
```

Comment:
These ABIs can be imported from lsp-smart-contracts lib. <https://github.com/lukso-network/lsp-smart-contracts/tree/develop/contracts>